cli/help.go

```diff
@@ -25,7 +25,8 @@ ARGUMENTS:
             {{printf "\t"}} * <ip1>,<ip2>{{end}}{{if .Flags}}
 OPTIONS:
    {{range .Flags}}{{.}}
-   {{end}}{{ end }}
+   {{end}}{{if (eq .Name "manage")}}{{printf "\t * swarm.overcommit=0.05\tovercommit to apply on resources"}}
```

Comment:
We don't have an option currently, but it's a bit bad to have the defaults duplicated. Out of scope with this PR.